src/lib/ai/mental-llama/routing/MentalHealthTaskRouterProduction.ts

```
@@ -0,0 +1,725 @@
+
+    logger.info('Using fallback classification', { failureReason });
+
+    // Try keyword-based fallback first
+    const keywordResult = this.matchKeywords(text);
+    if (keywordResult) {
+      return {
+        ...keywordResult,
+        method: 'keyword_fallback',
+        insights: {
+          ...keywordResult.insights,
+          fallbackReason,
```

ESLint

[error] 'fallbackReason' is not defined.

(no-undef)